ui/src/virtualconsole/vccuelist.cpp

```diff
@@ -819,10 +819,12 @@ void VCCueList::slotCurrentStepChanged(int stepNumber)
         if (slValue > 255)
             slValue = 255;
 
+        int upperBound = 255 - slValue;
+        int lowerBound = qFloor(upperBound - stepVal);
         //qDebug() << "Slider value:" << m_slider1->value() << "Step range:" << (255 - slValue) << (255 - slValue - stepVal);
         // if the Step slider is already in range, then do not set its value
         // this means a user interaction is going on, either with the mouse or external controller
-        if (m_slider1->value() < (255 - slValue - stepVal) || m_slider1->value() > (255 - slValue))
+        if (m_slider1->value() < lowerBound || m_slider1->value() >= upperBound)
         {
             m_slider1->blockSignals(true);
```

Comment:
Unrelated comment: This line can be rewritten to `const QSignalBlocker blocker(m_slider1);`; ` m_slider1->blockSignals(false);` down below should be removed.